### DISCUSSION TITLE
New Feature: Decimal to hysteresis

### DISCUSSION THREAD

**Discussion by Fr0stX76** [Rem0o/FanControl.Releases#563] [Ideas]
Salut Remi,

Would you consider adding decimals to hysteresis temp in graphs? With the nice addition (thanks!) of decimals to graphs, which has now given the option of granular control of water coolant temp for watercooled setup, I now see the 1c minimum hysteresis as being a bit on the high side. Being able to set it to 0.2 to 0.5 would be ideal.

Again, just a suggestion of feature for an already amazing tool!

Merci et bonne jounee

**Comment by Rem0o** [comment]
Salut!

Definitely a next step.  
There is still a few place that could use decimal places, like the linear fan curves (min temp, max temp, hysteresis).

Will do a review of all settings that are still integers for temperature, and put floats if possible, including hysteresis for the graph
